src/drivers/cmakeFileApi.ts

```diff
@@ -486,10 +487,8 @@ function convertToExtCodeModelFileGroup(targetObject: CodeModelKind.TargetObject
         if (sourceFile.compileGroupIndex !== undefined) {
             fileGroup[sourceFile.compileGroupIndex].sources.push(fileRelativePath);
         } else {
-            fileGroup[defaultIndex].sources.push(fileRelativePath);
-            if (!!sourceFile.isGenerated) {
-                fileGroup[defaultIndex].isGenerated = sourceFile.isGenerated;
-            }
+            const i = !!sourceFile.isGenerated ? generatedIndex : defaultIndex;
```

Comment:
Testing this, it looks like this fixes the issue. 

Right now, after debugging, it seems like we have the `isGenerated` flag at the `fileGroup` level, when they likely should be at the `sourceFile` level. 

I'm good with this fix in order to fix it for the user, but I think we should make a TODO for in the future, improving the code model interface and this code.